core/model/modx/processors/workspace/packages/uninstall.class.php

```diff
@@ -62,7 +62,7 @@ public function process() {
         );
 
         if ($this->package->uninstall($options) == false) {
-            return $this->failure(sprintf($this->modx->lexicon('package_err_uninstall'),$this->package->getPrimaryKey()));
+            return $this->failure(sprintf($this->modx->lexicon('package_err_uninstall',  array('signature' => $this->package->get('signature'))), $this->package->getPrimaryKey()));
```

Comment:
Let's use short array syntax for new code at least.

Reply:
You know, this is a controversial moment. Last time they explained to me that I need to use the current syntax. And I had to redo my changes.

Reply:
Let's just match the current syntax used in that file and we can worry about short array syntax in the 3.x branch.